pkg/kodo/cmd/build_test.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"fmt"
+	"testing"
+
+	cmp "github.com/google/go-cmp/cmp"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	buildv1api "github.com/openshift/api/build/v1"
+	imagev1api "github.com/openshift/api/image/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var envVar = EnvironmentVariables{
+	Namespace: "buildtest",
+}
+
+var deployVar = DeploymentVariables{
+	Source: "https://github.com/openshift/ruby-hello-world.git",
+}
+
+func getBuildConfig() buildv1api.BuildConfig {
+	return buildv1api.BuildConfig{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
So we can actually create a test case for createTypeMeta() and createObjectType(), this way we can use the functions directly in the getBuildConfig() in the tests, instead of repeating TypeMeta: metav1.TypeMeta{
			Kind:       "BuildConfig",
			APIVersion: "build.openshift.io/v1",
		},
		ObjectMeta: metav1.ObjectMeta{
			Name:      "my-app-docker-build",
			Namespace: "buildtest",
		},
twice, once in getBuildConfig() and getImageStream(), increases code reuse.

Reply:
Gotcha, working on it,

Reply:
done